example/gardener-local/calico/kustomization.yaml

```diff
@@ -2,4 +2,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 
 resources:
-- https://projectcalico.docs.tigera.io/v3.22/manifests/calico.yaml
+# Pin calico manifests to a specific version by copying released manifests to this repository.
+# With this, we are also not affected any more by downtimes of https://projectcalico.docs.tigera.io/v3.22/manifests/calico.yaml
+# which caused flakes of e2e test jobs in the past.
+- calico.yaml # extracted from https://github.com/projectcalico/calico/releases/download/v3.22.2/release-v3.22.2.tgz (manifests/calico.yaml)
```

Comment:
Why we have to copy the manifests? Isn't there a github repo hosting `manifests/calico.yaml` so that we ping the concrete tag?

Reply:
Nope, at least I couldn't find one. 
https://github.com/projectcalico/calico/blob/v3.22.2/calico/manifests/calico.yaml only contains some templates but no rendered manifests.

I wasn't able to find any rendered (and released) manifests other than in the release artifacts, unfortunately :/

Reply:
Should we go ahead like this to get the tests running, and eventually ask guys from calico if they would know of any such repositories?

Reply:
Sounds good

Reply:
> IMO, better approach would be to link the CRDs from https://github.com/projectcalico/calico/tree/v3.22.2/libcalico-go/config/crd:

Then we have pinned versions of the CRD manifests, but `https://projectcalico.docs.tigera.io/v3.22/manifests/calico.yaml` is still not pinned to a specific version.

Also, note that we have observed some flakes caused by short downtimes of `https://projectcalico.docs.tigera.io/` in the past few weeks.

Reply:
> eventually ask guys from calico if they would know of any such repositories?

Indeed, there doesn't seem to be a better way than what I did.